heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -198,9 +222,10 @@ void updateTopologyHandler(String topologyName, String newParallelism)
 
     LOG.fine("Sending Updating topology request: " + updateTopologyRequest);
     if (!schedulerClient.updateTopology(updateTopologyRequest)) {
-      throw new TopologyRuntimeManagementException(String.format(
+      throw new TopologyRuntimeManagementException(
           "Failed to update topology with Scheduler, updateTopologyRequest="
-              + updateTopologyRequest));
+              + updateTopologyRequest + "The topology can be in a strange stage. "
+                  + "Please check carefully or redeploy the topology !!");
```

Comment:
Do you need this another warning?

Reply:
The user usually assumes the topology keeps same as before the command is issued, however, the topology may be in unknown state. This warning is explicitly to tell user the check the topology state.